draft-ietf-quic-recovery.md

```diff
@@ -961,22 +961,22 @@ As an example of a well-known and publicly available implementation of a flow
 pacer, implementers are referred to the Fair Queue packet scheduler (fq qdisc)
 in Linux (3.11 onwards).
 
-## Restart after idle
-
-A connection is idle if there are no bytes in flight and there is no pending
-ack-eliciting data to send.  This can occur when the connection is
-application limited or after a probe timeout. In order to limit
-the size of bursts sent into the network, the behavior when restarting from
-idle depends upon whether pacing is used.
-
-If the sender uses pacing, the connection should limit the initial burst of
-packets to no more than the initial congestion window and subsequent packets
-SHOULD be paced. The congestion window does not change while the connection
-is idle.
-
-A sender that does not use pacing SHOULD reset its congestion window to the
-minimum of the current congestion window and the initial congestion window.
-This recommendation is based on Section 4.1 of {{?RFC5681}}.
+
+## Sending data after an idle period
+
+A sender becomes idle if it has no bytes in flight.  A sender can become idle
+when it is application limited and has no data to send, or when it marks packets
+as lost (and therefore not in flight).
```

Comment:
```suggestion
 as lost, and therefore not in flight.
```

Reply:
Can this also happen when all packets are acknowledged?  I'm not sure what the preferred behavior is in that case.  But the current text now properly captures the "my last PTO was acked and everything else was declared lost."

Reply:
Yes, that is a fine case ... this is what we've always had, the RTO text was incorrect. Basically the sender goes idle as soon as bytes in flight goes to 0. Maybe that's all we need to say. I've removed more text to make this clearer.